cloud/src/meta-service/codec.cpp

```diff
@@ -17,6 +17,8 @@
 
 #include "codec.h"
 
+#include <cstdint>
```

Comment:
why do we need this?

Reply:
gcc need this. otherwise unknown type:  uint64_t